src/App.tsx

```diff
@@ -1,9 +1,22 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  function isQueryIncludes(value: string) {
+    return value
+      .toLocaleLowerCase()
+      .includes(query.toLocaleLowerCase());
+  }
```

Comment:
you have to use somrthing like trim for good work of solution. but i had no chance to pass test with trim (

Reply:
i'd try to add trim anyways